.github/workflows/ci.yml

```diff
@@ -46,6 +46,8 @@ jobs:
 #        run: cargo test --workspace --all-features
       - name: No-default features
         run: cargo test --workspace --no-default-features
+      - name: History check
```

Comment:
Nit: A bit late to the party, but I don't think you need to run this in the test matrix.  I.e. we can just run this ONCE instead of on windows/osx/linux.

Otherwise LGTM - thanks for the fix!!!